File: CAPITULO 01/C++/ex1.cpp
#include <stdio.h>
int main()
{
	int n1,n2,n3,n4,soma;
	printf ("Digite um numero real e positivo: ");
	scanf ("%d%*c",&n1);
	printf ("Digite outro numero real e positivo : ");
	scanf("%d%*c",&n2);
	printf("Digite um terceiro numero real e positivo :");
	scanf("%d%*c",&n3);
	printf ("Digie um quarto numero real e positivo : ");
	scanf("%d%*c",&n4);
	soma = n1+n2+n3+n4;
	printf("A soma de de todos os numeros digitados e de %d",soma);
	return 0;
}

File: CAPITULO 01/Java/ex1.java
import java.io.*;
import java.util.*;
		//Primeiro exercicio do livro da porfessora Edilene fácil esse *-)
	public class ex1 {
 	 public static void main (String [] args){		
		int n1,n2,n3,n4,soma;
		Scanner dado;
		dado = new Scanner (System.in);
		System.out.println("Digite um numero inteiro e positivo:  ");
		n1 = dado.nextInt();
		System.out.println("Digite outro numero inteiro e positivo: ");
		n2 = dado.nextInt();
		System.out.println("Digite um terceiro numero inteiro e positivo ");
		n3 = dado.nextInt();
		System.out.println("Digite um quarto numero inteiro e positivo :");
		n4 = dado.nextInt();
		soma = n1+n2+n3+n4;
		System.out.println("A soma de todos os numeros que você digitou e de "+soma+" numeros");
		//System.exit(0);
		
   	 }
	}		
